src/devtools/client/webconsole/reducers/messages.js

```diff
@@ -1403,6 +1403,21 @@ function maybeSortVisibleMessages(state, sortWarningGroupMessage = false, timeSt
   // messages with execution points, as either we aren't replaying or haven't
   // seen any messages yet.
   if (state.hasExecutionPoints) {
+    state.visibleMessages = [...state.visibleMessages].sort((a, b) => {
+      const compared = compareBigInt(
+        messageExecutionPoint(state, a),
+        messageExecutionPoint(state, b)
+      );
+      if (compared < 0) {
+        return -1;
+      } else if (compared > 0) {
+        return 1;
+      } else {
+        const _a = messageCountSinceLastExecutionPoint(state, a);
+        const _b = messageCountSinceLastExecutionPoint(state, b);
+        return _a < _b ? -1 : _a > _b ? 1 : 0;
+      }
+    });
     state.visibleMessages = sortBy(
       state.visibleMessages,
       message => BigInt(messageExecutionPoint(state, message)),
```

Comment:
This sorting code still needs to be removed.

Reply:
D'oh!